rtos/include/rtos/Thread.h

```diff
@@ -131,7 +131,8 @@ class Thread : private mbed::NonCopyable<Thread> {
 
     /** Starts a thread executing the specified function.
       @param   task           function to be executed by this thread.
-      @return  status code that indicates the execution status of the function.
```

Comment:
In the commit message, replace "C++ exceptions diables" with "C++ exceptions disabled", and "valid runtime situition." with "valid runtime situation."